bundles/org.openhab.binding.sagercaster/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sagercaster" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="sagercaster">
+		<label>SagerCaster Thing</label>
+		<description>This thing represents a forecast for a given location</description>
+
+		<channels>
+			<channel id="forecast" typeId="forecast" />
+			<channel id="velocity" typeId="velocity" />
+			<channel id="velocity-beaufort" typeId="wind-speed-beaufort">
+				<label>Wind Velocity</label>
+			</channel>			
+			<channel id="wind-from" typeId="wind-direction">
+				<label>Wind from</label>
+			</channel>  
+			<channel id="wind-to" typeId="wind-direction">
+				<label>Wind to</label>
+			</channel>		   
```

Comment:
labels should have each word capitalized
```suggestion
			<channel id="wind-from" typeId="wind-direction">
				<label>Wind From</label>
			</channel>  
			<channel id="wind-to" typeId="wind-direction">
				<label>Wind To</label>
			</channel>		   
```
That said maybe these labels could be rewritten. Their purpose isn't clear as written, perhaps adding a description would help.